HHS/Head-Start-TTADP
# Add support for PIV card logins and disable debug logging

## Description of change
Adds support for the data format returned from the user-info endpoint from HSES for the PIV card user logins. After the HSES upgrade, while the username/password user logins worked, the PIV card users were not since the format of the data changed. This PR also points to the new HSES url for staging/dev/sandbox environments and disables the extra logging.


## How to test
On staging:
- go to https://tta-smarthub-staging.app.cloud.gov/
- if in possession of a PIV card, click on "Login with PIV"
- an error message will show
- after seeing the error message, use username and password to log in
- at this point you should see a button allowing connecting the PIV card
- click the above button
- logout
- login with PIV
- testing without PIV can be done by using the username and password to verify the classic login works
- verify that login into the hub works and the app behaves as expected

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-176


## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
